web/sites/default/settings.php

```diff
@@ -2,264 +2,73 @@
-        ],
-        'cache.backend.redis' => [
-          'class' => 'Drupal\redis\Cache\CacheBackendFactory',
-          'arguments' => ['@redis.factory', '@cache_tags_provider.container', '@serialization.phpserialize'],
-        ],
-        'cache.container' => [
-          'class' => '\Drupal\redis\Cache\PhpRedis',
-          'factory' => ['@cache.backend.redis', 'get'],
-          'arguments' => ['container'],
-        ],
-        'cache_tags_provider.container' => [
-          'class' => 'Drupal\redis\Cache\RedisCacheTagsChecksum',
-          'arguments' => ['@redis.factory'],
-        ],
-        'serialization.phpserialize' => [
-          'class' => 'Drupal\Component\Serialization\PhpSerialize',
-        ],
-      ],
-    ];
-  }
-}
-
-// Public, private and temporary files paths.
-if (getenv('LAGOON')) {
-  $settings['file_public_path'] = 'sites/default/files';
-  $settings['file_private_path'] = 'sites/default/files/private';
-  $settings['file_temporary_path'] = 'sites/default/files/private/tmp';
-}
-
-// ClamAV settings.
-$config['clamav.settings']['scan_mode'] = 1;
-$config['clamav.settings']['mode_executable']['executable_path'] = '/usr/bin/clamscan';
-
-// Enable shield.
-if (getenv('LAGOON') && (getenv('DRUPAL_SHIELD_USER') && getenv('DRUPAL_SHIELD_PASS'))) {
-  $config['shield.settings']['credential_provider'] = 'shield';
-  $config['shield.settings']['credentials']['shield']['user'] = getenv('DRUPAL_SHIELD_USER');
-  $config['shield.settings']['credentials']['shield']['pass'] = getenv('DRUPAL_SHIELD_PASS');
-}
-
-
-// Hash Salt
-if (getenv('LAGOON')) {
-  $settings['hash_salt'] = hash('sha256', getenv('LAGOON_PROJECT'));
-}
-
-// Loading settings for all environment types.
-if (file_exists(__DIR__ . '/all.settings.php')) {
-  include __DIR__ . '/all.settings.php';
-}
-
-// Environment specific settings files.
-if (getenv('LAGOON_ENVIRONMENT_TYPE')) {
-  if (file_exists(__DIR__ . '/' . getenv('LAGOON_ENVIRONMENT_TYPE') . '.settings.php')) {
-    include __DIR__ . '/' . getenv('LAGOON_ENVIRONMENT_TYPE') . '.settings.php';
-  }
+else {
+  include $govcms_includes . '/development.settings.php';
 }
 
-// Configuration path settings.
-$config_directories[CONFIG_SYNC_DIRECTORY] = '/app/config/default';
-$config_directories['dev'] = '/app/config/dev';
-
-// Last: this servers specific settings files.
-if (file_exists(__DIR__ . '/settings.local.php')) {
-  include __DIR__ . '/settings.local.php';
+// Project specific overrides.
+if (file_exists($app_root . '/sites/default/project.settings.php')) {
+  include $app_root . '/sites/default/project.settings.php';
 }
 
-// Stage file proxy URL from production URL.
-if (getenv('LAGOON_ENVIRONMENT_TYPE') != 'production') {
-
-  if (getenv('LAGOON_PROJECT')) {
-    $origin = 'https://nginx-' . getenv('LAGOON_PROJECT') . '-master.govcms.amazee.io';
-    $config['stage_file_proxy.settings']['origin'] = $origin;
-  }
-
-  if (getenv('STAGE_FILE_PROXY_URL')) {
```

Comment:
`STAGE_FILE_PROXY_URL` is still described in `.env.default` and is removed here.

I am curious as to why this code block s removed - it's an incredibly useful module for non-production development. If that is the business direction how will this functionality be adopted by Saas - assuming you wanted this to be adoptable by SaaS seamlessly, and if that is the business direction should the `.env.default` reference be removed?

Reply:
This moves to the scaffold 
https://github.com/govCMS/scaffold-tooling/blob/master/drupal/settings/lagoon.settings.php#L126-L128